Simplified/MyBooks/NYPLMyBooksDownloadCenter+AudiobooksDownload.swift

```diff
@@ -64,6 +70,26 @@ extension NYPLMyBooksDownloadCenter {
     }
   }
   
+  /// - Parameters:
+  ///   - audiobookManager: Audiobook manager responsible for the download mechanism
+  ///   - bookID: The book identifier for updating download status through delegate
+  ///   - isHighPriority: Only pass in `true` if user is opening the audiobook to listen
+  ///
+  ///   We use `AudiobookManager` instead of `Audiobook` here because `AudiobookManager`
+  ///   allow us to determine if download is needed or not before presenting the audiobook.
+  @objc func addAudiobookManagerToDownloader(audiobookManager: DefaultAudiobookManager,
```

Comment:
nit: we can avoid some naming repetition here.... it's still clear that the 1st param is the audiobook manager I think:
```
addAudiobookManagerToDownloader(_ audiobookManager: DefaultAudiobookManager, ...)
```

Reply:
Fixed